client/src/Dashboard/DashboardTitle.js

```diff
@@ -25,10 +26,35 @@ export default function DashboardTitle({ dates, userState, getDashboardData }) {
     }
   }, [dates, dateFilterValue])
 
+  useEffect(() => {
```

Comment:
@cjhaddad can you take a look at this useEffect and give @nneka-nu your input on this implementation, vs. using Tailwind or Ant Design in-built behavior?

Reply:
@katelovescode sure thing!

Reply:
@nneka-nu per kate's suggestion, I believe we can do this by using tailwind's responsive design breakpoints: https://tailwindcss.com/docs/responsive-design

Reply:
> @rebeccakarasiktw @cjhaddad @csprayregen I wasn't able to easily find a mockup for the XS size of the dashboard - can someone take a look at this screenshot and let @nneka-nu know if any changes need to be made?

Hi @nneka-nu could you also add some padding/spacing between the elements? Something like this:
![Dashboard-XS](https://user-images.githubusercontent.com/26929739/113428546-fc323300-93a4-11eb-8ffc-a645b2971307.png)

Thank you so much for doing this! This is awesome! 😄

Reply:
@rebeccakarasiktw No problem. I'll make the changes. Glad to help! :)